framework/source/class/qx/event/handler/DragDrop.js

```diff
@@ -665,6 +669,10 @@ qx.Class.define("qx.event.handler.DragDrop",
      */
     _getDelta : function(e)
     {
+      if (e.getDelta) {
+        return e.getDelta();
+      }
+      
       if (!this.__startConfig) {
```

Comment:
who supports getDelta ? this also seems to be unrelated to the iframe fix

Reply:
this is supporte by the http://www.qooxdoo.org/devel/api/#qx.event.type.Track event, which was registerd above.